web/init/src/components/shared/Linter.jsx

```diff
@@ -4,12 +4,13 @@ import get from "lodash/get";
 
 export default class Linter extends React.Component {
 
-  constructor() {
-    super();
-    this.state = {
-      showHelp: false,
-      showPreview: true
-    };
+  constructor(props) {
+    super(props);
+
+    // if there's a readme, default to readme tab. Else default to linter results.
+    const showHelp = !this.props.readme;
+    const showPreview = !showHelp;
+    this.state = { showHelp, showPreview };
```

Comment:
This was not directly related but I don't think it makes sense to default to the readme tab if its empty (which happens in non init-chart workflows).